### PR TITLE
added UI

### PR DESCRIPTION
**Pull request by samvelArm**

### CODE REVIEW

**Comment by song-hao** [comment]
looks pretty nice, probably add a section in `README.md` about how to start the frontend server. something like: 
```
# Start frontend server:
$ npm install && npm start
# Goto http://localhost:3000/
```

**Comment by srujann** [comment]
@song-hao no need to start a frontend server separately. I am going submit a change to run this inside the java server.

File: shopping/src/main/ui/src/components/Card/Card.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            <div className="card clr-col-6 clr-col-sm-6 clr-col-md-4 clr-col-lg-3 clr-col-xl-2">
+                <div className="card-header">
+                    {item.name}
+                </div>
+                <div className="card-block image-container">
+                    <img src={item.src} alt="item.name"/>
+                </div>
+                <div className="card-footer">
+                    <div className="clr-row">
+                        <select
```
Comment (song-hao) [inline shopping/src/main/ui/src/components/Card/Card.js]: Maybe using some nice looking 3rd-party selection component. We tried multiple devices, seems Android phone having the problem as follows:

File: shopping/src/main/ui/src/App.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        src: image12,
+    }
+]
+
+class App extends PureComponent {
+    render() {
+        return (
+            <div className="application">
+                <div className="application-header">
+                    <h1>Beachshirts Application</h1>
```
Comment (song-hao) [inline shopping/src/main/ui/src/App.js]: @srujann Do you also want to change this to "3D Beachops" or just the name of the tab?

Comment (srujann) [reply shopping/src/main/ui/src/App.js]: @song-hao just tab name is fine for now.

**Comment by srujann** [comment]
lgtm

**Pull request merged**